operators/src/plot/statistics.rs

```diff
@@ -263,7 +271,7 @@ mod tests {
             sources: vec![raster_source].into(),
         };
 
-        let execution_context = MockExecutionContext::test_default();
+        let execution_context = MockExecutionContext::new_with_tiling_spec(tiling_specification);
```

Comment:
It's correct to do this. If we have to replace every test default, the default is… not a default anymore.

Reply:
yep but there are also tests that use the current default. i see no good solution here.